WrightTools/data/_colors.py

```diff
@@ -186,12 +182,7 @@ def from_COLORS(filepaths, name=None, cols=None, invert_d1=True,
             grid_i = griddata(points, zi, xi, method='linear', fill_value=fill_value)
             data.create_channel(name=key, values=grid_i)
     # axes
-    for expression in scanned.keys():
-        if 'w' in expression:
-            units = 'nm'
-        elif 'd' in expression:
-            units = 'fs'
-        data.create_axis(expression, units)
+    data.transform(scanned.keys())
```

Comment:
Units? Looks like you do something in transform, but want to make sure that those changes are for this purpose.